src/python_testing/taglist_and_topology_test_support.py

```diff
@@ -143,12 +144,16 @@ def create_device_type_list_for_root(direct_children, endpoint_dict: dict[int, A
 
 
 def cmp_tag_list(a: Clusters.Descriptor.Structs.SemanticTagStruct, b: Clusters.Descriptor.Structs.SemanticTagStruct):
+    if type(a.mfgCode) != type(b.mfgCode):
+        return -1 if type(a.mfgCode) is Nullable else 1
```

Comment:
in various places we seem to have `is NullValue` which seems clearer than checking the type. Could we use that?

![image](https://github.com/user-attachments/assets/9b6d74cf-273e-4a92-a1ca-96657b721919)

Reply:
That does become rather longer then because I need to check if a is null and b is not, or if b is null and a is not. Whereas what I really want is to know whether I can direct compare these.